src/Runner.Worker/ExecutionContext.cs

```diff
@@ -743,12 +744,19 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
             var base64EncodedToken = Convert.ToBase64String(Encoding.UTF8.GetBytes($"x-access-token:{githubAccessToken}"));
             HostContext.SecretMasker.AddValue(base64EncodedToken);
             var githubJob = Global.Variables.Get("system.github.job");
+            var resultsReceiverEndpoint = Global.Variables.Get("system.github.results_endpoint");
             var githubContext = new GitHubContext();
             githubContext["token"] = githubAccessToken;
+
             if (!string.IsNullOrEmpty(githubJob))
             {
                 githubContext["job"] = new StringContextData(githubJob);
             }
+            if (!string.IsNullOrEmpty(resultsReceiverEndpoint))
+            {
+                githubContext["resultsReceiverEndpoint"] = new StringContextData(resultsReceiverEndpoint);
```

Comment:
why do you need to expose `resultsReceiverEndpoint` as a github context?
Do we plan to support customers to use an expression with it? `${{github.resultsReceiverEndpoint}}`?

Reply:
Is that what the github context is used for? Good possibility we won't need to, we were just trying to determine where the right place to add it was